ports/libgourou/vcpkg.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "libgourou",
+  "version": "0.8.0",
+  "description": "libgourou is a free implementation of Adobe's ADEPT protocol used to add DRM on ePub/PDF files",
+  "homepage": "https://indefero.soutade.fr/p/libgourou",
+  "license": "LGPL-3.0-or-later",
+  "supports": "!uwp & !windows",
```

```suggestion
  "supports": "!windows",
```
`!windows` already contains `!uwp`.